### ISSUE TITLE
Doesn't work on Windows

### ISSUE THREAD

**Issue by xmha97** [insanum/sncli#119]
Doesn't work on Windows :(
![image](https://user-images.githubusercontent.com/23485114/152763524-5634f32b-bf59-4279-9f22-be3bf8934c73.png)

**Comment by swalladge** [comment]
@xmha97 yes that's correct, Windows is not supported.  It's not easy to add support either, because the UI library we use doesn't support Windows: https://github.com/urwid/urwid/issues/152

**Comment by N0ury** [comment]
But it works fine on WSL

**Comment by swalladge** [comment]
@N0ury yep, because WSL is a Linux variant. ;)

Maybe we should document supported systems in the readme; PRs welcome. :)

**Comment by xmha97** [comment]
OK, please write in the README file that is not working in Windows.
